pkg/storage/storagewrappers/combinedtuplereader.go

```diff
@@ -52,17 +53,21 @@ func (c *combinedTupleReader) Read(
 	return storage.NewCombinedIterator(iter1, iter2), nil
 }
 
+// ReadPage functions similarly to Read but includes support for pagination. It takes additional
+// pagination parameters and returns a slice of tuples along with a continuation token, which may
+// not be empty. This token can be used for retrieving subsequent pages of data.
 func (c *combinedTupleReader) ReadPage(
 	ctx context.Context,
 	store string,
 	tk *openfgav1.TupleKey,
 	opts storage.PaginationOptions,
 ) ([]*openfgav1.Tuple, []byte, error) {
-	// no reading from contextual tuples
+	// No reading from contextual tuples.
 
 	return c.RelationshipTupleReader.ReadPage(ctx, store, tk, opts)
 }
 
+// ReadUserTuple tries to return one tuple that matches the provided key exactly.
 func (c *combinedTupleReader) ReadUserTuple(
```

Comment:
Let's not duplicate docs here. All the methods here are already documented in `storage`. We should do just

```
// ReadUserTuple please see [[storage.RelationshipTupleReader.ReadUserTuple]]
```